unix/xdebug

```diff
@@ -1,11 +1,23 @@
-#!/bin/bash
+#!/usr/bin/env bash
+
+if [ -f "${HOME}/.config/xdebug-wrapper" ]; then
+    . "${HOME}/.config/xdebug-wrapper"
+fi
 
 # Update according to your needs
-XDEBUG_FILE=php_xdebug-2.4.0-7.0-vc14.dll
-XDEBUG_PATH=/path/to/
-PHP_BINARY=/usr/bin/php
-XDEBUG_IDEKEY=PHPSTORM
+PHP_BINARY=${PHP_BINARY:="php"}
+
+if ! [ "${EXTENSION_PATH}" ]; then
+    EXTENSION_PATH=$(${PHP_BINARY} -i | awk '/^extension_dir => (.*)$/ { print $3 }')
```

Comment:
At least on my build of PHP7 when you use it with ```-i``` it's format is ```key => value```. Is it different on Windows?